Object storage: fix acceptance tests for etags check

The issue occurred already thrice:
- https://github.com/terraform-providers/terraform-provider-openstack/pull/577#issuecomment-453442675
- https://github.com/terraform-providers/terraform-provider-openstack/pull/515#issuecomment-445744862
- https://github.com/terraform-providers/terraform-provider-openstack/pull/574#issuecomment-453477506

```
--- FAIL: TestAccObjectStorageV1Object_objectManifest (1.43s)
    testing.go:518: Step 0 error: Check failed: Check 3/3 error: openstack_objectstorage_object_v1.myfile: Attribute 'etag' expected "\"a84ffba5c71c24627a369146227d701a\"", got "\"207c91804d4cbf698be032d2dd7ff735\""
```

The reason is a race condition. See the golang playground [here](https://play.golang.org/p/s_1mdSUUkNY)

--- a/openstack/resource_openstack_objectstorage_object_v1_test.go
+++ b/openstack/resource_openstack_objectstorage_object_v1_test.go
@@ -406,5 +406,10 @@ resource "openstack_objectstorage_object_v1" "myfile" {
   name = "terraform/test.csv"
   container_name = "${openstack_objectstorage_container_v1.container_1.name}"
   object_manifest = "${format("%s/terraform/test.csv/part",openstack_objectstorage_container_v1.container_1.name)}"
+
+  metadata {
+    race = "${openstack_objectstorage_object_v1.myfile_part1.id}"
+    condition = "${openstack_objectstorage_object_v1.myfile_part2.id}"
+  }
 }
 `

